src/software/jetson_nano/services/motor.cpp

```diff
@@ -334,6 +336,15 @@ MotorService::MotorFaultIndicator MotorService::checkDriverFault(uint8_t motor)
     return MotorFaultIndicator(drive_enabled, motor_faults);
 }
 
+void MotorService::disableVelocity(double current_velocity, double previous_velocity)
+{
+    if (std::abs(current_velocity - previous_velocity) > RUNAWAY_PROTECTION_THRESHOLD_MPS)
+    {
+        driver_control_enable_gpio_.setValue(GpioState::LOW);
+        LOG(FATAL) << "Front right motor runaway";
```

Comment:
you should make this fatal message specific to the motor that crashed

Reply:
Removing the function as it is unused